src/services/azure/constants.rs

```diff
@@ -0,0 +1,15 @@
+// Env values used in azure services.
+//AZURE_STORAGE_ACCOUNT = Credential.access_name
+#[allow(dead_code)]
+pub const AZURE_STORAGE_ACCOUNT: &str = "AZURE_STORAGE_ACCOUNT";
```

Comment:
BTW, does azure has documented the behavior about `AZURE_STORAGE_ACCOUNT`?

Based on https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azblob, it seems that azure storage uses `AZURE_STORAGE_ACCOUNT_NAME` and `AZURE_STORAGE_ACCOUNT_KEY` instead.

Reply:
I started out with this variable name from [here](https://github.com/Azure/azure-sdk-for-rust/blob/main/sdk/storage_blobs/examples/blob_00.rs) I'll double check, it's not good to mix too many variable names together

Reply:
It seems that Azure storage doesn't have the unify env behavior here. How about removing those support for now?

- Golang: use `AZURE_STORAGE_ACCOUNT_NAME`, `AZURE_STORAGE_ACCOUNT_KEY`
- Python: None
- Rust: `STORAGE_ACCOUNT` without `azure` prefix which is not suitable for reqsign.

Reply:
AZURE_STORAGE_ACCOUNT_NAME  and AZURE_STORAGE_ACCOUNT_KEY make more sense

Reply:
`AZURE_STORAGE_ACCOUNT_NAME` and `AZURE_STORAGE_ACCOUNT_KEY` makes sense to me too.